gerrit-entrypoint.sh

```diff
@@ -185,6 +185,8 @@ if [ "$1" = "/gerrit-start.sh" ]; then
   su-exec ${GERRIT_USER} java ${JAVA_OPTIONS} ${JAVA_MEM_OPTIONS} -jar "${GERRIT_WAR}" init --batch -d "${GERRIT_SITE}" ${GERRIT_INIT_ARGS}
   if [ $? -eq 0 ]; then
     echo "Reindexing..."
+    su-exec ${GERRIT_USER} java ${JAVA_OPTIONS} ${JAVA_MEM_OPTIONS} -jar "${GERRIT_WAR}" reindex --verbose -d "${GERRIT_SITE}"
```

Comment:
Well the server didn't boot when upgrading (I only had a tiny test repo on the server). I did check and you are correct that it now indexes on every boot regardless of whether there was an upgrade or not. 
I guess the simple answer to that would be to write the current gerrit version into a file and then compare the file against the env var on boot to figure out whether there was an upgrade.
The alternative would be to somehow auto start the online indexer (although I only glanced over #46 so I am not sure whether this would help at this early stage).